ampersand-select-view.js

```diff
@@ -104,18 +104,25 @@ SelectView.prototype.onChange = function () {
     var value = this.select.options[this.select.selectedIndex].value;
 
     if (this.options.isCollection && this.yieldModel) {
-        value = this.findModelForId(value);
+        value = this.getModelForId(value);
     }
 
     this.setValue(value);
 };
 
-SelectView.prototype.findModelForId = function (id) {
```

Comment:
renamed to `get` prefix to fall inline with other methods.  made function `0` friendly